kani-driver/src/args/cargo.rs

```diff
@@ -94,9 +94,9 @@ impl ValidateArgs for CargoCommonArgs {
     }
 }
 
-/// Arguments that Kani pass down into Cargo test essentially uninterpreted.
+/// Arguments that cargo Kani supports to select build / test target.
```

Comment:
Adding a link to https://doc.rust-lang.org/cargo/commands/cargo-test.html#target-selection to clarify what is meant by `target` could be useful.